Source/SwiftLintFramework/Rules/ModifiersOrderRule.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  ModifiersOrderRule.swift
+//  SwiftLint
+//
+//  Created by Jose Cheyo Jimenez on 05/06/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+// TODO: Get somehow `SwiftDeclarationAttribute` for Swift 4.1
```

Comment:
class and static? class can be an attribute or a keyword

Reply:
added `TODO`

Reply:
This issue is painful. I will push an update in the following days, but `static` and `class` is not recognised as attributes by `SourceKitten`

Reply:
can You file a radar?

Reply:
I have added a workaround in my latest commit, but its an ugly hack. Please let me know what do you think, or we should wait until a further release. 

I will open a radar meanwhile.

Reply:
Actually, now that i am just about to open the radar, i can already see, add them as `source.lang.swift.decl.var.static` or `source.lang.swift.decl.function.method.class`. They might just answer it is not a declaration attribute, but a declaration.  Maybe i could work out something based on that, rather than this regex search i am doing now. However, somehow i should still fake them i am affraid..

For example:
```
class Foo { 
   static public let bar = 3
}
```

will give

```swift
{
"key.diagnostic_stage": "source.diagnostic.stage.swift.parse", 
"key.substructure": [
  { 
    "key.bodylength": 31, 
    "key.nameoffset": 6, 
     "key.accessibility": "source.lang.swift.accessibility.internal", 
     "key.length": 43, 
     "key.substructure": [
          {
               "key.nameoffset": 32, 
               "key.accessibility": "source.lang.swift.accessibility.public", 
               "key.length": 25, 
               "key.name": "bar", 
               "key.kind": "source.lang.swift.decl.var.static", 
               "key.namelength": 3, 
               "key.offset": 14, 
               "key.attributes": [
                    { 
                         "key.attribute": "source.decl.attribute.public", 
                         "key.offset": 21, 
                         "key.length": 6}]
                    }
     ], 
     "key.runtime_name": "_TtC8__main__3Foo", 
     "key.name": "Foo", 
     "key.kind": "source.lang.swift.decl.class", 
     "key.namelength": 3, 
     "key.offset": 0, 
     "key.bodyoffset": 11
     }
],
"key.offset": 0, 
"key.length": 43
}
```

Reply:
Meanwhile, i decided to raise the issue, lets see what they say.
https://bugs.swift.org/browse/SR-7252